keyboards/kibou/suisei/info.json

```diff
@@ -0,0 +1,101 @@
+{
+    "manufacturer": "kibou",
+    "keyboard_name": "suisei",
+    "maintainer": "kibou",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B2", "A10", "A9", "A8", "B15", "B14", "B13", "A15", "B3", "B4", "B5", "B6", "B7", "B8", "C13", "B9"],
+        "rows": ["B11", "B10", "B0", "A1", "A0"]
+    },
+    "processor": "STM32F072",
+    "url": "https://kibou.store/",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0065",
+        "vid": "0x586A"
+    },
+    "indicators": {
+        "caps_lock": "B1"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This is a 65% keyboard?  

If it matches the 65% layout in the community layouts, support for that should be added.

Reply:
Apparently the board layout is WKL with no WK option, and we don't have any WKL community layouts at the moment:
![layout](https://github.com/qmk/qmk_firmware/assets/616082/6af571a7-29b6-48dd-9233-e57ab92c91da)
The PCB image at the [shop page](https://kibou.store/products/suisei-addons) also does not show any other layout options for the bottom row; however, there is an option to use either a 2u or a 2×1u split backspace.  If this is actually the case, the keyboard should have two layouts defined — maybe `LAYOUT_65_ansi_wkl` and `LAYOUT_65_ansi_wkl_split_bs`, and the `LAYOUT_all` alias pointing to `LAYOUT_65_ansi_wkl_split_bs`.

Reply:
> 

Yes, this is a 65% keyboard